README.md

```diff
@@ -73,12 +71,17 @@ For running locally, you should ensure
   Alternatively, these may be configured using `~/.aws/credentials`.
   If you add a new profile to `~/.aws/credentials`, you can then tell the server to use it by setting `AWS_PROFILE=...`.
 
+If you have built the next.js part of the site (`./static-site`), via `npx next build static-site` or similar, then setting `NEXT_JS_PRODUCTION=1` will use these built assets rather than server-side compilation.
```

Comment:
@victorlin - adding a thread here as you wanted to look into `NEXT_JS_PRODUCTION` - this variable is only in play for local development (it allows you to use the next.js statically generated assets). When `NODE_ENV=production`, e.g. the heroku review apps, this variable isn't used. We could avoid this if it were simple to run the server in production mode locally, but I find that near impossible (ssl redirect, session secret keys, redis keys etc etc); and I didn't want to complicate this PR with making it easier to run in production locally.

Reply:
@victorlin are you still looking into this part? Happy to jump on a call and answer any questions if so

Reply:
I just tried the npm scripts locally and understand better now.

Having a 2nd "production" flag was a bit confusing to me. How about renaming to something that describes the behavior more literally, such as `USE_PREBUILT_STATIC_SITE`?

There's a slight change in that `server.js`, `npm run server`, and `npm run groups` will no longer use prebuilt static-site files by default. I think this preferable with static-site effectively adhering to the values `NODE_ENV=development` vs. `NODE_ENV=production` unless this new flag is set.

Reply:
👍 [Updated](https://github.com/nextstrain/nextstrain.org/compare/a237a3fe289d1df150a88b3f3c005c55c4f54c51..c078420a277bc15c78aacd8fd04c52fda9c79293) to use `USE_PREBUILT_STATIC_SITE`

> There's a slight change in that server.js, npm run server, and npm run groups will no longer use prebuilt static-site files by default. 

Right, but since [node.js runs in "development" mode by default](https://nodejs.org/en/learn/getting-started/nodejs-the-difference-between-development-and-production#nodejs-the-difference-between-development-and-production) I'd argue that this is more in line with expectations and the previous situation was strange. I do think we should make it easier to run in "production" mode locally (not with the same secrets we actually use in production of course!).

Reply:
Agreed - sorry if it wasn't clear in the last sentence. Thanks for the rename!

One more small request related to this: could you move this snippet from the README into infrastructure.md's [Environment (or config) variables section](https://docs.nextstrain.org/projects/nextstrain-dot-org/en/latest/infrastructure.html#environment-or-config-variables)? I realize that `GATSBY_DEV_URL` didn't have an entry but I think it should've and this new env var should too.

https://github.com/nextstrain/nextstrain.org/blob/c078420a277bc15c78aacd8fd04c52fda9c79293/README.md?plain=1#L74

Reply:
Totally! [Done as part of this force-push](https://github.com/nextstrain/nextstrain.org/compare/c078420a277bc15c78aacd8fd04c52fda9c79293..c673b1be652297ab0c112c755f8ba140569c5214)